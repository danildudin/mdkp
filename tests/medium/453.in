29 8 0
425 106 915 855 140 637 186 439 659 808 23 551 69 861 854 627 323 947 354 801 584 670 877 421 197 426 677 424 134 
367 103 567 544 799 264 427 760 993 22 695 544 901 807 795 363 273 319 440 956 49 253 434 738 76 788 455 587 6 
444 703 474 553 430 535 305 951 570 404 730 188 356 731 505 248 206 409 318 894 218 847 376 415 313 921 347 642 912 
248 328 459 318 494 395 551 90 316 15 624 110 107 16 281 489 669 659 169 405 498 605 293 254 305 272 143 210 147 
945 527 619 314 420 781 926 564 577 909 873 516 391 866 649 252 960 260 668 465 358 493 338 408 433 259 362 785 786 
352 13 547 209 574 43 459 877 438 949 822 353 854 463 172 942 575 722 328 925 938 211 66 729 66 42 688 890 978 
760 898 308 782 454 359 976 432 656 240 233 475 915 348 400 137 577 133 337 790 419 413 743 438 320 925 757 937 948 
744 912 406 452 628 998 294 645 120 74 718 344 475 214 321 556 863 613 538 854 808 460 350 247 37 143 729 489 941 
253 727 838 422 336 787 53 732 233 878 738 282 403 948 622 631 627 936 521 798 618 69 259 406 64 821 127 759 222 
6342 9428 6140 6279 7362 5650 9699 6250 
