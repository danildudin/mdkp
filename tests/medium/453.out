9301
12
2 3 8 9 11 13 14 15 17 21 22 26 
